index.css

```diff
@@ -0,0 +1,132 @@
+:root {
+  font-size: 13px;
+}
+
+.navbar {
+  position: fixed;
+  width: 100%;
+  z-index: 1;
+  padding: 5px 30px;
+}
+
+.new-font {
+  font-family: "Audiowide", cursive;
+}
+.content-container {
+  width: 100%;
+  height: 100%;
+}
+.gradient-font {
```

Comment:
Use variable for colors

Reply:
Do i have to use variables if im only going to use it at one place